runtime/sema/string_type.go

```diff
@@ -152,6 +158,13 @@ It does not modify the original string.
 If either of the parameters are out of the bounds of the string, or the indices are invalid (` + "`from > upTo`" + `), then the function will fail
 `
 
+const StringTypeReplaceAllFunctionName = "replaceAll"
+const StringTypeReplaceAllFunctionDocString = `
+Returns a new string after replacing all the occurrences of parameter ` + "`of` with the parameter `with`" + `.
+
+If with is empty, it matches at the beginning of the string and after each UTF-8 sequence, yielding up to k+1 replacements for a k-rune string.
```

Comment:
I am not 100% on whether we should use the term `rune` in the public documentation. Open to suggestions on a better term.

Reply:
We could say "... yielding up to k+1 replacements for a string of length `k`"

Reply:
Thanks, makes more sense. I removed the `up to` part since it'll be exactly `k + 1` for an empty `with` param.